Releases/Manifests/6.34.0.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Firebase" : "6.34.0",
+  "FirebaseCore" : "6.10.4",
+  "FirebaseCrashlytics" : "4.6.2",
+  "FirebaseFirestore" : "1.19.0",
+  "FirebaseInstanceID" : "4.8.0",
```

Comment:
Installations needs to be added as well since that was in the cherry-pick.

Reply:
Done